content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,185 @@
+enaled by setting the `PILOT_ENABLE_CROSS_CLUSTER_WORKLOAD_ENTRY` environment variable in istiod.
+  ([Issue #29026](https://github.com/istio/istio/issues/29026))
+
+- **Added** `istioctl analyze` now informs if deprecated or alpha-level annotations are present.
+(These checks can be disabled using `--suppress "IST0135=*"` and `--suppress "IST0136=*"`
+respectively.)
+  ([Issue #29154](https://github.com/istio/istio/issues/29154))
+
+- **Added** option to enable STS token fetch and exchange for XDS flow.
+  ([Issue #29943](https://github.com/istio/istio/issues/29943))
+
+- **Added** OIDC JWT authenticator that supports both JWKS-URI and OIDC discovery. The OIDC JWT authenticator will be used when configured through the JWT_RULE env variable.  ([Issue #30295](https://github.com/istio/istio/issues/30295))
+
+- **Added** a new experimental proxy option, `DNS_AUTO_ALLOCATE`, to control auto allocation of ServiceEntry addresses. Previously,
+this option was tied to `DNS_CAPTURE`. Now, `DNS_CAPTURE` can be enabled without auto allocation.
+  ([Issue #29324](https://github.com/istio/istio/issues/29324))
+
+- **Added** support for backpressure on XDS pushes to avoid overloading Envoy during periods of high configuration
+churn. This is disabled by default and can be enabled by setting the PILOT_ENABLE_FLOW_CONTROL environment variable in Istiod.
+  ([Issue #25685](https://github.com/istio/istio/issues/25685))
+
+- **Added** Istio Grafana Dashboards Query Reporter Dropdown.
+  ([Issue #27595](https://github.com/istio/istio/issues/27595))
+
+- **Added** `istioctl x injector list` command to show which namespaces have Istio sidecar injection
+and, for control plane canaries, show all Istio injectors and the namespaces they control.
+  ([Issue #23892](https://github.com/istio/istio/issues/23892))
+
+- **Added** The locality load balancing docs have been re-written into a
+formal traffic management task. The new docs describe in more detail
+how locality load balancing works as well as how to configure both
+failover and weighted distribution. In addition, the new docs are now
+automatically verified for correctness. ([Pull Request #29651](https://github.com/istio/istio/pull/29651))
+
+- **Added** Istio 1.9 supports Kubernetes versions 1.17 to 1.20.
+  ([Issue #30176](https://github.com/istio/istio/issues/30176))
+
+- **Added** support of PeerAuthentication per-port-level configuration on pass through filter chains.
+  ([Issue #27994](https://github.com/istio/istio/issues/27994))
+
+- **Added** support for applications that bind to their pod IP address, rather than wildcard or localhost address, through the `Sidecar` API. ([Pull Request #28178](https://github.com/istio/istio/pull/28178))
+
+- **Added** Correctly iptables rules and listener filters setting to support original src ip preserve within a cluster.  ([Issue #23369](https://github.com/istio/istio/issues/23369))
```

Comment:
Is this feature only avail in TPROXY mode @hzxuzhonghu ?

Reply:
yes